src/agentscope/studio/static/js/dashboard-detail.js

```diff
@@ -16,7 +16,7 @@ function initializeDashboardDetailPageByUrl(pageUrl) {
 }
 
 // The dashboard detail page supports three tabs:
-// 1. dialogue tab: the dialogue history of the runtime instance
+// 1. dialogue tab: the conversation history of the runtime instance
```

Comment:
We change all  the word `dialogue` into `conversation`, so maybe named as conversation tab?

Reply:
Done